styles/style.css

```
@@ -22,128 +15,127 @@
-  background-size: 100% auto;
+  background-color: #4b4b4b;
+  background-image: url("../img/jpg/index-background.jpg");
+  background-size: 100%;
   background-repeat: no-repeat;
 }
-.form {
-  background-color: #e5e5e5;
-  color: #ffffff;
-  background-image: url("../img/backgroundform.jpg");
-  background-size: 100% auto;
-  background-repeat: no-repeat;
+.image-vector {
+  text-align: center;
 }
-.catalog-products {
-  background-color: #e5e5e5;
-  color: #ffffff;
-  background-image: url("../img/catalog-background.jpg");
-  background-size: 100% auto;
-  background-repeat: no-repeat;
+.header-text{
+font-weight: bold;
+font-size: 30px;
+line-height: 36px;
+text-align: center;
+text-transform: uppercase;
 }
+.text-main{
+  font-size: 22px;
+  line-height: 36px;
+  text-align: center;
+}
+
+/*catalog block*/
 .advantages-item-white {
   background-color: #83b3d3;
-  color: #ffffff;
 }
 .advantages-describe-list {
   background-color: #ffffff;
 }
 .advantages-describe-item-rgb12 {
-  background-color: rgba(131, 179, 211, 0.12);
+  background-color: rgba(131, 179, 211, 0.12),#83B3D3;
 }
 .advantages-describe-item-rgb120 {
-  background-color: rgba(131, 179, 211, 0.2);
+  background-color: rgba(131, 179, 211, 0.2),#83B3D3;
 }
 
-.navigation-link-header {
-  font-size: 16px;
-  line-height: 21px;
-  text-transform: uppercase;
-  font-weight: bold;
-  color: #000000;
-  text-decoration: none;
-}
-.navigation-link-header {
-  background-color: #756257;
-  color: #ffffff;
-  font-size: 16px;
-  line-height: 21px;
+.catalog-header{
   font-weight: bold;
+  font-size: 24px;
+  line-height: 28px;
+  text-align: center;
   text-transform: uppercase;
 }
-.description-main {
-  font-weight: bold;
-  font-size: 30px;
-  line-height: 36px;
-}
-.text-main {
-  font-size: 22px;
-  line-height: 36px;
+.catalog-text{
+text-align: center
 }
-.advantages-title-white {
+.catalog-title-white{
   font-weight: bold;
   font-size: 24px;
   line-height: 28px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #FFFFFF;
 }
-.advantages-describe-title {
-  font-weight: bold;
-  font-size: 24px;
-  line-height: 28px;
+.catalog-text-white{
+  text-align: center;
+  color: #FFFFFF;
 }
-.search-item {
-  background-color: #756257;
+/*form-block*/
+.form {
+  background-color: #e5e5e5;
   color: #ffffff;
-  font-size: 20px;
-  line-height: 36px;
+  background-image: url("../img/jpg/backgroundform.jpg");
+  background-size: 100% auto;
+  background-repeat: no-repeat;
+}
+.search-describe{
+  font-size: 22px;
+  line-height: 26px;
+  text-align: center;
 }
 .form-title {
   font-weight: bold;
   font-size: 30px;
   line-height: 36px;
+  text-align: center;
+  text-transform: uppercase;
 }
 .form-text {
   font-size: 22px;
   line-height: 26px;
+  text-align: center;
+}
+.form-user{
+  text-align: center;
 }
 .button-form {
   background-color: #83b3d3;
   color: #ffffff;
   font-size: 20px;
   line-height: 26px;
```

```suggestion
  line-height: 26px;
  font-family: inherit;
  font-weight: bold;
  text-transform: uppercase;
```
Ну либо на этом элементе добавить класс `button`
Но  `font-family: inherit;` все равно нужен, потому что шрифты у кнопок по дефолту не наследуются

---

Вроде сделала